content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -270,9 +282,15 @@ CPU cores).
 ## Checkpoint experiments
 
 To track successive steps in a longer or deeper <abbr>experiment</abbr>, you can
-register [checkpoints](/doc/user-guide/experiment-management/checkpoints) from
-your code. Running checkpoint experiments is no different than with regular
-ones, e.g.:
+register "checkpoints" from your code. These combine DVC Experiments with code
+logging. The latter can be achieved either with [DVCLive](/doc/dvclive), by
+using `dvc.api.make_checkpoint()` (Python code), or writing signal files (any
+programming language) following the same steps as `make_checkpoint()`.
```

Comment:
Note: for now I mentioned the different ways to run checkpoints in the Running Exps guide.